sdks/python/apache_beam/runners/dataflow/dataflow_runner_test.py

```diff
@@ -206,7 +206,7 @@ def test_create_runner(self):
 
   def test_environment_override_translation(self):
     self.default_properties.append('--experiments=beam_fn_api')
-    self.default_properties.append('--worker_harness_container_image=FOO')
```

Comment:
Probably add a new unit test for "sdk_container_image" instead to make sure "worker_harness_container_image" works as well (at least for runner v1).